lib/static/modules/reducers/tree/helpers.js

```diff
@@ -110,9 +110,9 @@ export function updateImagesStatus({tree, view, images, imageIdsArray, newStatus
             .map(imageId => getUpdatedProperty(tree, diff, ['images', 'byId', imageId, 'status']));
 
         const finalStatus = determineFinalStatus(imageStatuses);
-        const trueResultStatus = isAssertViewError(result.error) && finalStatus === ERROR ? FAIL : finalStatus;
+        const trueResultStatus = isImageError(result.error) && finalStatus === ERROR ? FAIL : finalStatus;
 
-        if (isAssertViewError(result.error) && trueResultStatus !== result.status) {
+        if (isImageError(result.error) && trueResultStatus !== result.status) {
```

Comment:
In Testplane, we have a contract: if test result has image errors, test result has "AssertViewError", and images can have either `ImageDiffError` or `NoRefImageError`.

In Playwright adapter, though, we can have `ImageDiffError` or `NoRefImageError` as result.error. Looks like we could encounter other issues because of that.